riscv/overlap_list.h

```diff
@@ -21,3 +21,4 @@ DECLARE_OVERLAP_INSN(rstsa16, EXT_ZPN)
 DECLARE_OVERLAP_INSN(rstsa32, EXT_ZPN)
 DECLARE_OVERLAP_INSN(srli32_u, EXT_ZPN)
 DECLARE_OVERLAP_INSN(umax32, EXT_ZPN)
+DECLARE_OVERLAP_INSN(mop_r_28, EXT_ZIMOP)
```

Comment:
I realized we should also declare overlap for mop_rr_7 which is also used by Zicfiss.